internal/git/git.go

```diff
@@ -265,23 +258,10 @@ func locateServiceReleaseRollbackSkipCondition(env, service string, n uint) cond
 		}
 		n--
 		return false
-	}
-}
-
-func locateServiceRollbackCondition(env, service string) conditionFunc {
-	r := regexp.MustCompile(fmt.Sprintf(`(?i)\[%s/%s] rollback `, regexp.QuoteMeta(env), regexp.QuoteMeta(service)))
-	return func(commitMsg string) bool {
-		if env == "" {
-			return false
-		}
-		if service == "" {
-			return false
-		}
-		return r.MatchString(commitMsg)
-	}
+	})
 }
 
-type conditionFunc func(commitMsg string) bool
+type conditionFunc = func(commitMsg string) bool
```

Comment:
What is the use case of this type alias? They are unusual (though part of the language) so would love to understand :)